app/controllers/front_ui_controller.rb

```diff
@@ -18,17 +18,15 @@ def all_maps
   end
 
   def nearby_mappers
-    @nearby_maps = []
-
-    if current_location.present?
-      lat = session[:lat]
-      lon = session[:lon]
-      @nearby_maps = Map.maps_nearby(lat: lat, lon: lon, dist: 10)
-                        .page(params[:maps])
-                        .per_page(12)
-    end
-
-    @all_mappers = Map.featured_authors.paginate(page: params[:mappers], per_page: 12)
+    return unless current_location.present?
```

Comment:
maybe we need to test this too?

Reply:
Oh yeah, makes sense...adding the test. Thanks Jeff